pitmutationmate-override-plugin/plugin/src/main/groovy/io/github/amosproj/pitmutationmate/override/strategy/GradlePitestPluginOverrideStrategy.groovy

```diff
@@ -64,4 +75,15 @@ class GradlePitestPluginOverrideStrategy implements OverrideStrategy {
         log.debug("Property '$propertyName' successfully overwritten with '$newValue'.")
     }
 
+    void addPitestDependency(Project project, String overrideValue) {
+        try {
+            project.subprojects {
+                buildscript {
+                    dependencies.add('pitest', overrideValue)
+                }
+            }
+        } catch (ConfigurationNotFoundException e) {
+            log.debug('Tried to add the dependency to all subprojects in buildscript.dependencies ' + e)
```

Comment:
The message seems very vague to me. Hasn't the overriding part failed now? Shouldn't there be a info or even warning when that happens?